From: Alessio Treglia <alessio@debian.org>
Date: Fri, 4 Jun 2010 03:12:00 +0200
Subject: Drop GCC options which are not supported on all architectures.

Author: Alessio Treglia <alessio@debian.org>
Origin: Debian
Bug: http://bugs.debian.org/584492
Forwarded: yes
---
 src/CMakeLists.txt |    4 ++++
 1 file changed, 4 insertions(+)

--- yoshimi.orig/src/CMakeLists.txt
+++ yoshimi/src/CMakeLists.txt
@@ -130,6 +130,9 @@
 set (BuildOptionsRelease
     "-O3" CACHE STRING "Compile with optimisation for Release builds")
 
+set (BuildOptionsDebian
+    "-O3 -ffast-math -fomit-frame-pointer -ldl" CACHE STRING "Debian build flags")
+
 find_package (PkgConfig REQUIRED)
 if (PKG_CONFIG_FOUND)
     message(STATUS "Found pkg-config ${PKG_CONFIG_EXECUTABLE}")
@@ -331,8 +334,8 @@
     message (STATUS "DEBUG Build, flags: ${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_DEBUG}")
 else (BuildForDebug)
     set (CMAKE_BUILD_TYPE "Release")
-    set (CMAKE_CXX_FLAGS_RELEASE ${BuildOptionsRelease})
-    message (STATUS "RELEASE Build, flags: ${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_RELEASE}")
+    set (CMAKE_CXX_FLAGS_RELEASE ${BuildOptionsDebian})
+    message (STATUS "RELEASE/Debian Build, flags: ${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_RELEASE}")
 endif (BuildForDebug)
 
 configure_file(
