From: Alessio Treglia <alessio@debian.org>
Date: Fri, 4 Jun 2010 03:12:00 +0200
Subject: Drop GCC options which are not supported on all architectures.

Author: Alessio Treglia <alessio@debian.org>
Origin: Debian
Bug: http://bugs.debian.org/584492
Forwarded: yes
---
 src/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1687492..a510364 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -124,6 +124,9 @@ set (BuildOptionsBasic
 set (BuildOptionsDebug
     "-O0 -g3 -ggdb -Wall -Wpointer-arith" CACHE STRING "Debug build flags")
 
+set (BuildOptionsDebian
+    "-O3 -ffast-math -fomit-frame-pointer -ldl" CACHE STRING "Debian build flags")
+
 find_package (PkgConfig REQUIRED)
 if (PKG_CONFIG_FOUND)
     message(STATUS "Found pkg-config ${PKG_CONFIG_EXECUTABLE}")
@@ -327,6 +330,7 @@ else (BuildForDebug)
             endif (BuildForCore2_X86_64)
         endif (BuildForAMD_X86_64)
     endif (BuildFor0ld_X86)
+    set (CMAKE_CXX_FLAGS_RELEASE ${BuildOptionsDebian})
     message (STATUS "Building for ${CMAKE_BUILD_TYPE}, flags: ${CMAKE_CXX_FLAGS_RELEASE}")
 endif (BuildForDebug)
 
