Description: Fix FTBFS on hurd-i386: PATH_MAX undeclared.
Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=885249
Author: Svante Signell <svante.signell@gmail.com>
Forwarded: Will Godfrey <willgodfrey@musically.me.uk>

Index: yoshimi/src/Misc/Config.cpp
===================================================================
--- yoshimi.orig/src/Misc/Config.cpp
+++ yoshimi/src/Misc/Config.cpp
@@ -231,17 +231,16 @@ bool Config::Setup(int argc, char **argv
     }
     if (restoreState)
     {
-        char * fp;
+        char *fp = NULL;
         if (! StateFile.size()) goto no_state0;
-        else fp = new char [PATH_MAX];
+        fp = realpath (StateFile.c_str(), NULL);
+        if (fp == NULL) goto no_state0;
 
-        if (! realpath (StateFile.c_str(), fp)) goto no_state1;
         StateFile = fp;
-        delete (fp);
+        free (fp);
 
         if (! isRegFile(StateFile))
         {
-            no_state1: delete (fp);
             no_state0: Log("Invalid state file specified for restore " + StateFile, 2);
             return true;
         }
Index: yoshimi/src/Misc/MiscFuncs.cpp
===================================================================
--- yoshimi.orig/src/Misc/MiscFuncs.cpp
+++ yoshimi/src/Misc/MiscFuncs.cpp
@@ -339,9 +339,9 @@ string MiscFuncs::setExtension(string fn
 // replace 'src' with a different one in the compilation directory
 string MiscFuncs::localPath(string leaf)
 {
-    char *tmpath;
-    tmpath = (char*) malloc(PATH_MAX);
-    getcwd (tmpath, PATH_MAX);
+    char *tmpath = getcwd (NULL, 0);
+    if (tmpath == NULL)
+       return "";
     string path = (string) tmpath;
     size_t found = path.rfind("/src");
     if (found != string::npos)
