Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Origin: Debian
Description: Fix passing LDFLAGS.
Forwarded: yes

Index: yoshimi/src/CMakeLists.txt
===================================================================
--- yoshimi.orig/src/CMakeLists.txt	2014-03-20 16:55:52.531720116 +0100
+++ yoshimi/src/CMakeLists.txt	2014-03-20 16:58:09.692457955 +0100
@@ -362,11 +362,11 @@
     z
 )
 
-set (CMAKE_EXE_LINKER_FLAGS "-ldl")
+#set (CMAKE_EXE_LINKER_FLAGS "-ldl")
 
 add_executable (yoshimi ${ProgSources} main.cpp)
 
-target_link_libraries (yoshimi ${ExternLibraries})
+target_link_libraries (yoshimi ${ExternLibraries} "-dl")
 
 install (TARGETS yoshimi RUNTIME DESTINATION bin)
 
